include/rdma/fabric.h

```diff
@@ -340,6 +340,7 @@ struct fi_fabric_attr {
 	char			*prov_name;
 	uint32_t		prov_version;
 	uint32_t		api_version;
+	char			*comp_list;
```

Comment:
comp_list is acting as a filter on input to fi_getinfo, making it really a provider usable field only.  It is usable on output for component selection.  Could be used to indicate whether an implementation is native or software emulated.  Need to decide on final string format, for example, can we capture implementation details for the lowest provider as to which implementation is being used?